LithoStrings/Classes/NSMutableAttributeStringAdds.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  NSMutableAttributeStringAdds.swift
+//  LithoStrings
+//
+//  Created by Remmington Damper on 1/26/22.
+//
+
+import Foundation
+import UIKit
+
+    public func setForegroundColorAttributes(color: UIColor, string: NSMutableAttributedString) -> Void {
+        var attributes: [NSAttributedString.Key:Any] = [:]
+        addForegroundColorToAttributes(color: color, attributes: &attributes)
+        string.addAttributes(attributes, range: NSMakeRange(0, string.length))
```

Comment:
we don't need to define `attributes` or add the foregroundColor to it. You can instead just use the function you made `returnForegroundColor` directly in line 14

Reply:
One of the functions was meant to be used with server code. That's why we have it there.

Reply:
Just because it's not used internally in the rest of the project doesn't mean it won't be useful for other projects, (which it will be for us). If this can be accomplished in less lines and be just as readable, we should do that.

Reply:
@cjc8 I'm getting errors about calling a non-function type when I try to take out the other 2 lines. Swift seems to be complaining because the method signature doesn't work. Here's a screenshot.

<img width="1440" alt="Screen Shot 2022-02-24 at 12 25 40 PM" src="https://user-images.githubusercontent.com/81644534/155575708-0c0eccee-ab84-46a5-aa01-581346190ffd.png">